app/parse.py

```diff
@@ -15,8 +27,57 @@ class Product:
     num_of_reviews: int
 
 
+def get_products_from_page(driver: WebDriver, url: str) -> list[Product]:
+    driver.get(url)
+    time.sleep(1)
+    driver.refresh()
+    while True:
+        try:
+            button = driver.find_element(By.CLASS_NAME, "ecomerce-items-scroll-more")
+        except NoSuchElementException:
+            break
+        if button.get_attribute("style") == "display: none;":
+            break
+        button.click()
+        time.sleep(1)
+    products_soup = driver.find_elements(By.CLASS_NAME, "product-wrapper")
+    products = []
+    for product_soup in products_soup:
+        products.append(
+            Product(
+                title=product_soup.find_element(By.CLASS_NAME, "title").get_attribute("title"),
+                description=product_soup.find_element(By.CLASS_NAME, "description").text,
+                price=float(product_soup.find_element(By.CLASS_NAME, "price").text[1:]),
```

Comment:
The code uses product_soup.find_element(By.CLASS_NAME, "price").text[1:] to parse the price, assuming that the price text always starts with a currency symbol. This may fail for other currencies or formats. It’s better to account for different scenarios.